modules/core/shared/src/main/scala/scaladex/core/api/SearchEndpoints.scala

```diff
@@ -6,7 +6,7 @@ import endpoints4s.algebra
 // There definition is shared here, for consistency.
 trait SearchEndpoints extends algebra.Endpoints with algebra.JsonEntitiesFromSchemas {
 
-  implicit val response: JsonSchema[AutocompletionResponse] =
+  implicit val autocompletionResponse: JsonSchema[AutocompletionResponse] =
```

Comment:
Renamed this for consistency, along with `params`.

Since `ApiDocumentation` extends both `SearchEndpoints` and `ArtifactEndpoint`, there would be an issue if both have fields named `param` and `response`.

Reply:
:+1